app/cdap/components/ConnectionList/Components/TabLabelCanBrowse/index.tsx

```diff
@@ -55,7 +46,7 @@ export default function TabLabelCanBrowse({
       title={label}
       arrow
       key={`tooltip-${index}`}
-      data-testid="connections-tab-ref-label-browse"
+      data-testid={'connections-tab-ref-label-browse' + index}
```

Comment:
we should not use indices in test ids and ids
plz use label
convert label to lowercase, and replace spaces with dashes

Reply:
done